src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub type ReportCollection<'a> = Vec<&'a Report>;
 pub struct ReportFooter {
     pub message: String,
     pub notes: Vec<String>,
+    pub with_summary: bool,
```

Comment:
```suggestion
    pub summary: bool,
```
 we shouldn't add a new method on footer that allows you to change this to false.